lib/baconmail/digest.rb

```diff
@@ -9,98 +9,138 @@ def to_imap_date
 
 module Baconmail
   class Digest
-    def self.daily_digest(account, configs)
-      log         = Logger.new(STDOUT)
-      emails      = []
-      aib_domain  = account.username.split("@")[1]
-    
+    attr_reader :account, :configs
+
+    def initialize(account, configs)
+      @account = account
+      @configs = configs
+    end
+
+    def daily_digest
       log.info("Account: #{account.username}")
-      Gmail.new(account.username, account.password) do |gmail|
-        gmail.mailbox('[Gmail]/All Mail').emails(:on => (Date.today - 1)).each do |email|
-          next if email.subject.match("Daily Digest for") rescue nil
-          next if "#{email.from[0]['mailbox']}@#{email.from[0]['host']}" == account.username
-            
-          mailbox = email[:to][0].mailbox.downcase rescue "unknown"
 
-          next if Baconmail::Settings.instance.instance_variable_get(:@blacklist).include?(mailbox)
```

Comment:
@caffo could you clarify where does `@blacklist` come from?

Reply:
There was a missing setting on the repo. I updated master with it.